.github/workflows/main.yml

```
@@ -0,0 +1 @@
+
```

actionlint (1.7.10)

workflow is empty

(syntax-check)